.github/workflows/ci-actions.yml

```diff
@@ -89,6 +89,13 @@ jobs:
           path: ~/.m2/repository
           # refresh cache every month to avoid unlimited growth
           key: q2maven-${{ steps.get-date.outputs.date }}
+      # We do this so we can get better analytics
+      - name: Update Docker Client User Agent
+        shell: bash
+        run: |
+          cat ~/.docker/config.json
```

Comment:
I just added the `cat` commands for debugging. 

If `sed` works properly, I will remove them